llm_toolkit/models.py

```diff
@@ -350,6 +350,16 @@ class GeminiPro(GeminiModel):
   _vertex_ai_model = 'gemini-1.0-pro'
 
 
+class GeminiExperimental(VertexAIModel):
+  """Gemini Experimental, results maybe genius or delightfully unpredictable."""
+
+  _max_output_tokens = 8192
+  context_window = 32760
```

Comment:
Not a big deal for our current workloads, but is this actually the same 1M context windows as Gemini 1.5?

Reply:
Done